tldw_Server_API/app/core/Watchlists/alert_rules.py

```
@@ -0,0 +1,244 @@
+"""
+Watchlist Alert Rules — Evaluation Engine
+
+Evaluates user-defined alert rules against completed watchlist run statistics
+and creates notifications when conditions are met.
+
+Supported condition types:
+- no_items: Run produced zero items
+- error_rate_above: Error rate exceeds threshold (0.0–1.0)
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)